Content.Server/StationEvents/BasicStationEventSchedulerSystem.cs

```diff
@@ -20,11 +21,20 @@ public sealed class BasicStationEventSchedulerSystem : GameRuleSystem<BasicStati
     {
         [Dependency] private readonly IRobustRandom _random = default!;
         [Dependency] private readonly EventManagerSystem _event = default!;
+        [Dependency] private readonly IGameTiming _timing = default!;
+
+        protected override void Started(EntityUid uid, BasicStationEventSchedulerComponent eventComp, GameRuleComponent gameRule, GameRuleStartedEvent args)
+        {
+            base.Started(uid, eventComp, gameRule, args);
+
+            ResetTimer(eventComp);
+            eventComp.NextEventTime += eventComp.MinimumTimeUntilFirstEvent;
```

Comment:
While I appreciate the code reuse, this technically makes the earliest time for the first event `MinimumTimeBetweenEvents + MinimumTimeUntilFirstEvent`, doesn't it?

Reply:
It does, which I note in the Component's docs. I considered this a bit of a non-issue, but it does technically mean events will wait a *little* further into a round before triggering. If this is a concern the new datafield can easily be modified :P